_pages/2019/x/psets/0/scratch.md

```diff
@@ -32,13 +27,15 @@ It's time to choose your own adventure! Your assignment, quite simply, is to imp
 * Your project must use at least one sound.
 * Your project should be more complex than most of those demonstrated in lecture (many of which, though instructive, were quite short) but it can be less complex than *Ivy's Hardest Game*. As such, your project should probably use a few dozen puzzle pieces overall.
 
-Here are your predecessors' [projects from last year](https://scratch.mit.edu/studios/4248580/), though implemented in Scratch 2. But try to think of an idea on your own, and then set out to implement it. However, don't try to implement the entirety of your project all at once: pluck off one piece at a time. In other words, take baby steps: write a bit of code (i.e., drag and drop a few puzzle pieces), test, write a bit more, test, and so forth. And select **File > Download to your computer** every few minutes so that you don't lose any work!
+Head to [the Scratch website](https://scratch.mit.edu/) and click the orange "Start Creating" button to begin. 
```

Comment:
We should probably suggest that folks sign up for an account so that their work is autosaved now.

Reply:
Have added the signing up step.